src/Guide/Api/Methods.hs

```diff
@@ -30,23 +31,23 @@ import qualified Guide.Search as Search
 ----------------------------------------------------------------------------
 
 -- | Get a list of available categories.
-getCategories :: DB -> Handler [CCategoryInfo]
+getCategories :: DB -> Guider [CCategoryInfo]
 getCategories db = do
   dbQuery db GetCategories <&> \xs ->
     map toCCategoryInfo xs
 
 -- | Get a single category and all of its items.
-getCategory :: DB -> Uid Category -> Handler CCategoryFull
+getCategory :: DB -> Uid Category -> Guider CCategoryFull
 getCategory db catId = toCCategoryFull <$> getCategoryOrFail db catId
 
 -- | Create a new category, given the title and the grandparent (aka group).
 --
 -- Returns the ID of the created category (or of the existing one if the
 -- category with this title exists already).
-createCategory :: DB -> Text -> Text -> Handler (Uid Category)
+createCategory :: DB -> Text -> Text -> Guider (Uid Category)
 createCategory db title' group' = do
-  when (T.null title') $ do throwError (err400 {errBody = "Title not provided"})
-  when (T.null group') $ do throwError (err400 {errBody = "Group' not provided"})
+  when (T.null title') $ throwError err400 {errBody = "Title not provided"}
```

Comment:
If you decide to remove parentheses, perhaps it's better to make the coupling tighter so as not to confuse people.

```suggestion
  when (T.null title') $ throwError err400{errBody = "Title not provided"}
```